api/tests/opentrons/protocol_engine/commands/magnetic_module/test_engage.py

```diff
@@ -1,37 +1,32 @@
 """Test magnetic module engage commands."""
 
-import pytest
 from decoy import Decoy
 
-from opentrons.protocol_engine.execution import (
-    EquipmentHandler,
-    MovementHandler,
-    PipettingHandler,
-    RunControlHandler,
+from opentrons.protocol_engine.execution import EquipmentHandler
+from opentrons.protocol_engine.commands.magnetic_module import (
+    EngageParams,
+    EngageResult,
 )
-from opentrons.protocol_engine.commands.magnetic_module import EngageParams
 from opentrons.protocol_engine.commands.magnetic_module.engage import (
     EngageImplementation,
 )
 
 
-@pytest.mark.xfail(strict=True, raises=NotImplementedError)
 async def test_magnetic_module_engage_implementation(
     decoy: Decoy,
     equipment: EquipmentHandler,
-    movement: MovementHandler,
-    pipetting: PipettingHandler,
-    run_control: RunControlHandler,
 ) -> None:
-    """It should engage the magnets."""
-    subject = EngageImplementation(
-        equipment=equipment,
-        movement=movement,
-        pipetting=pipetting,
-        run_control=run_control,
-    )
+    """It should delegate to the equipment handler."""
+    subject = EngageImplementation(equipment=equipment)
     params = EngageParams(
         moduleId="module-id",
         engageHeight=3.14159,
     )
-    await subject.execute(params=params)
+    result = await subject.execute(params=params)
+    decoy.verify(
+        await equipment.engage_magnets(
+            magnetic_module_id="module-id",
+            mm_above_labware_base=3.14159,
+        )
+    )
+    assert result == EngageResult()
```

Comment:
If we returned data, this could look like:

```suggestion
    decoy.when(
        await equipment.engage_magnets(
            magnetic_module_id="module-id",
            mm_above_labware_base=3.14159,
        )
    ).then_return(42)

    result = await subject.execute(params=params)

    assert result == EngageResult(mmFromBase=42)
```